Userland/Libraries/LibWeb/Streams/WritableStreamDefaultController.cpp

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2023, Matthew Olsson <mattco98@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibWeb/Streams/WritableStream.h>
+#include <LibWeb/Streams/WritableStreamDefaultController.h>
+
+namespace Web::Streams {
+
+// https://streams.spec.whatwg.org/#close-sentinel
+// Note: This is a non-standard function that assists in implementing the above "unique value". Implements the "close sentinel" value.
+JS::Value create_close_sentinel()
+{
+    // The close sentinel is a unique value enqueued into [[queue]], in lieu of a chunk, to signal that the stream is closed. It is only used internally, and is never exposed to web developers.
+    // Note: We use the empty Value to signal this as, similarly to the note above, the empty value is not exposed nor creatable to web developers.
+    return {};
```

Comment:
Not a huge fan of adding more uses of the empty value type but I guess it is the easiest solution here. Just makes removing it harder.